napari/layers/shapes/shapes.py

```diff
@@ -2141,6 +2134,14 @@ def expand_shape(self, data):
         data_full : array
             Full D dimensional data array of the shape.
         """
+        warnings.warn(
```

Comment:
do you want to deprecated in the docstring as well, so that it's shown in the documentation ?

Reply:
Done, can you let me know if I've done the formatting right. Thanks!!

Reply:
(both here and above in `displayed_coordinates`)